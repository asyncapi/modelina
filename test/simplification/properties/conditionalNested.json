{
    "properties": { 
      "testProp1": { 
        "type": "string"
      }
    },
    "if": {
        "properties": { 
          "testProp1": { 
            "const": "string"
          } 
        } 
    },
    "then": {
        "properties": {
          "testProp2": { 
            "type": "string"
          } 
        },
        "if": {
            "properties": { 
              "testProp2": { 
                "const": "string"
              } 
            } 
        },
        "then": {
            "properties": { 
              "testProp3": { 
                "type": "string"
              } 
            } 
        },
        "else": {
            "properties": { 
              "testProp4": { 
                "type": "string"
              } 
            } 
        }
    },
    "else": {
        "properties": { 
          "testProp5": { 
            "type": "string"
          } 
        } 
    }
}