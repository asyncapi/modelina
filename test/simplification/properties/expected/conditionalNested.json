{
  "testProp1": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  },
  "testProp2": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  },
  "testProp3": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  },
  "testProp4": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  },
  "testProp5": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  }
}